examples/k8s_minikube/README.md

```diff
@@ -0,0 +1,83 @@
+
+Test RabbitMQ-Autoclsuter on K8s
+
+ 1. Install [`kubectl`](https://kubernetes.io/docs/tasks/kubectl/install/): 
+```
+# OS X
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/darwin/amd64/kubectl
+
+# Linux
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+
+chmod +x ./kubectl
+sudo mv ./kubectl /usr/local/bin/kubectl
+```
+2.Install [`Minikube`](https://github.com/kubernetes/minikube/releases):
+
+```
+OSX
+
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.17.1/minikube-darwin-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
+
+Linux
+
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.17.1/minikube-linux-amd64 && chmod +x minikube && sudo mv minikube /usr/local/bin/
+```
+
+3. Start `minikube` virtual machine:
+`minikube start --cpus=2 --memory=2040 --vm-driver=virtualbox` 
+
+4. Create a namespace only for RabbitMQ test:
+`kubectl create namespace test-rabbitmq`
+
+5. Run the `etcd` image and expose it:
+```
+kubectl run etcd --image=microbox/etcd --port=4001 --namespace=test-rabbitmq -- --name etcd
+kubectl --namespace=test-rabbitmq expose deployment etcd
+```
+6. Build a [Docker image](https://github.com/rabbitmq/rabbitmq-autocluster/blob/master/Dockerfile)
+```
+git clone https://github.com/rabbitmq/rabbitmq-autocluster.git rabbitmq-autocluster
+make dist
```

Comment:
Here the users have to build the plugin for source. 
We can remove:
```
git clone https://github.com/rabbitmq/rabbitmq-autocluster.git rabbitmq-autocluster
make dist
```
when we will put the `docker` image on docker hub